packages/devtools_app/lib/src/inspector/layout_explorer/layout_explorer.dart

```diff
@@ -30,9 +32,11 @@ class _LayoutExplorerTabState extends State<LayoutExplorerTab>
     if (BoxLayoutExplorerWidget.shouldDisplay(node)) {
       return BoxLayoutExplorerWidget(controller);
     }
-    return const Center(
+    return Center(
       child: Text(
-        'Currently, Layout Explorer only supports Box and Flex-based widgets.',
+        node != null
+            ? 'Currently, Layout Explorer only supports Box and Flex-based widgets.'
+            : 'Select a widget to view its layout.',
```

Comment:
We could also leave the layout explorer window completely blank when no widget is selected. @InMatrix @cobblest any thoughts?

Reply:
Or should we always select the root if no widget is selected?

Reply:
Leave the layout explorer window there could help discovery. As an enhancement to the current message, can we suggest a widget to inspect? For example, the second line can be modified to 'Try it out: select the first `Row` widget to view its layout.`  [Go button].

Reply:
I like the suggestion of finding the first Row widget. Just need to know what alternate message to show if there are no Row/Column widgets in the app.

Reply:
How about linking to online documentation in that case? We could also consider embedding a short video demo in that space.